pkg/parser/parser.go

```diff
@@ -51,7 +51,11 @@ func (e Error) String() string {
 
 func New(input string, builtins map[string]*FuncDecl) *Parser {
 	l := lexer.New(input)
-	p := &Parser{funcs: builtins, wssStack: []bool{false}}
+
+	p := &Parser{funcs: map[string]*FuncDecl{}, wssStack: []bool{false}}
+	for name, funcDecl := range builtins {
```

Comment:
:pineapple: shame you can't use generics - you could use `x/exp/maps.Clone()` here (or on the line before even: `funcs: maps.Clone(builtins)`)

Reply:
oh nice!! i think tinygo 1.27.0 is coming out soon and it works with got 1.20.
I wonder if the issue was again that I didn't hoist things on main.
I've got a cleanup PR due that replaces all errormessages with Printfs now that that's possible. In fact, I should maybe consider exposing them in evy!?